BrainPortal/app/views/tool_configs/_by_resource.erb

```diff
@@ -97,7 +97,7 @@
         <th><%= for_tool ? "Execution Servers" : "Tools" %> </th>
         <th>Versions Configured</th>
         <th>Projects In Effect</th>
-        <th class="left_align">Users with access problem(s)</th>
+        <th class="left_align">Users with access problem(s) out of <%= users.count %></th>
```

Comment:
I'd rather have the column named the constant `Users Access Summary` and then the column filled with messages like on of these three:

```
(No user access) # in red
```

```
(All users) # in green
```

```
23 users have access, 56 users cannot access  # in green then red
```

Reply:
ok